src/app/app.component.scss

```diff
@@ -1,16 +1,23 @@
 body,
 html {
-  height: 100%;
+  min-height: 100%;
   margin: 0;
 }
 
+.content {
+  flex: 1 1 auto;
+  width: 100%;
+}
+
 .cent {
-  min-height: 100vh;
   display: flex;
   flex-direction: column;
-  align-items: stretch;
   justify-content: center;
-  background-color: #CCE8F2;
-  place-items: center;
-
+  align-items: center;
```

Comment:
this field should be stretch to leave the box go with the content, the way you done it will make the text or content go out from his box in my experience